src/Akka.Streams.Kafka.Tests/KafkaFixture.cs

```diff
@@ -0,0 +1,165 @@
+            {
+                Name = _networkName
+            });
+            await _client.Networks.ConnectNetworkAsync(network.ID, new NetworkConnectParameters()
+            {
+                Container = _kafkaContainerName
+            });
+            await _client.Networks.ConnectNetworkAsync(network.ID, new NetworkConnectParameters()
+            {
+                Container = _zookeeperContainerName
+            });
+
+            // start the containers
+            await _client.Containers.StartContainerAsync(_zookeeperContainerName, new ContainerStartParameters());
+            await _client.Containers.StartContainerAsync(_kafkaContainerName, new ContainerStartParameters());
+
+            // Provide a 10 second startup delay
+            await Task.Delay(TimeSpan.FromSeconds(10));
+        }
+
+        public async Task DisposeAsync()
+        {
+            if (_client != null)
+            {
+                // Stop Kafka
+                await _client.Containers.StopContainerAsync(_kafkaContainerName, new ContainerStopParameters());
+                await _client.Containers.RemoveContainerAsync(_kafkaContainerName, new ContainerRemoveParameters { Force = true });
+                
+                // Stop Zookeeper
+                await _client.Containers.StopContainerAsync(_zookeeperContainerName, new ContainerStopParameters());
+                await _client.Containers.RemoveContainerAsync(_zookeeperContainerName, new ContainerRemoveParameters { Force = true });
+
+                // Delete network between containers
+                await _client.Networks.DeleteNetworkAsync(_networkName);
+                
+                _client.Dispose();
+            }
+        }
+        
+        private async Task CreateContainer(string imageName, string imageTag, string containerName, int portToExpose, Dictionary<string, string> env)
+        {
+            await _client.Containers.CreateContainerAsync(new CreateContainerParameters
+            {
+                Image = $"{imageName}:{imageTag}",
+                Name = containerName,
+                Tty = true,
+                
+                ExposedPorts = new Dictionary<string, EmptyStruct>
+                {
+                    {$"{portToExpose}/tcp", new EmptyStruct()}
+                },
+                HostConfig = new HostConfig
+                {
+                    PortBindings = new Dictionary<string, IList<PortBinding>>
+                    {
+                        {
+                            $"{portToExpose}/tcp",
+                            new List<PortBinding>
+                            {
+                                new PortBinding
+                                {
+                                    HostPort = $"{portToExpose}"
+                                }
+                            }
+                        }
+                    },
+                    ExtraHosts = new [] { "moby:127.0.0.1", "localhost:127.0.0.1" },
```

Comment:
What's the `moby:127.0.0.1` for? Is this for Docker on Windows?

Reply:
Mm, this is something I just took from docker-compose.yml file that was used before to make setup in the same way it was working before. I do not see any reference about it in official image documentation, so will try to remove it and check tests passing in separate PR.